packs/nitropack.js

```diff
@@ -0,0 +1,41 @@
+  'modern-image-formats': 'Use [`Image Optimization`](https://support.nitropack.io/hc/en-us/articles/16547237162513) to automatically convert your images to WebP and improve your loading times.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Automatic Image Lazy Loading` is the name of a feature and becomes link text to additional documentation.*/
+  'offscreen-images': 'Defer offscreen images by enabling [`Automatic Image Lazy Loading`](https://support.nitropack.io/hc/en-us/articles/12457493524369-NitroPack-Lazy-Loading-Feature-for-Images). This will load images only when they need to be displayed on user’s screen, resulting in quicker page load times.',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Image Optimization`, `Minify HTML`, `Font Subsetting`, `Reduce Unused CSS` are names of features and become links to additional documentation.*/
+  'total-byte-weight': 'To reduce network payloads, enable the following NitroPack settings that help with the compression and lazy loading of your resources, including — [`Image Optimization`](https://support.nitropack.io/hc/en-us/articles/16547237162513); [`Minify HTML`](https://support.nitropack.io/hc/en-us/articles/360061059394-Minify-Resources); [`Font Subsetting`](https://support.nitropack.io/hc/en-us/articles/10172273306769); [`Reduce Unused CSS`](https://support.nitropack.io/hc/en-us/articles/360020418457-Reduce-Unused-CSS); [`Font Compression Upgrade`](https://support.nitropack.io/hc/en-us/articles/16547272270865).',
+  /** Additional description of a Lighthouse audit for a third-party framework called `NitroPack`. This is displayed after a user expands the section to see more. No character length limits. `Remove render-blocking resources`, `Delayed Scripts`, `Font Subsetting`, `Optimized CSS Delivery` and `Reduce Unused CSS` are names of features and become links to additional documentation.*/
+  'render-blocking-resources': 'Enable [`Remove render-blocking resources`](https://support.nitropack.io/hc/en-us/articles/13820893500049-How-to-Deal-with-Render-Blocking-Resources-in-NitroPack) and [`Delayed Scripts`](https://support.nitropack.io/hc/en-us/articles/1500002600942-Delayed-Scripts) in NitroPack for faster initial load times. For even better results, activate [`Font Subsetting`](https://support.nitropack.io/hc/en-us/articles/10172273306769), [`Optimize CSS Delivery`](https://support.nitropack.io/hc/en-us/articles/16547344670353) and [`Reduce Unused CSS`](https://support.nitropack.io/hc/en-us/articles/360020418457-Reduce-Unused-CSS) – these features will optimize your CSS and font files so that they don’t block the rendering of your page.',
```

Comment:
Please only use 1 link per item. If there are multiple features you want to highlight, I would suggest linking to them in your own documentation.